README.md

```diff
@@ -315,7 +324,7 @@ If you want to anyway run the parallel tests sequentially. You should use --forc
 ```commandline
 python -u testrunner.py -te TEST-17412  -tp TEST-18382 --target s3-vm-2928 --force_serial_run True
 ```
-If you want to run your test plan and test execution ticket with test runner, you should skip --force_serial_run switch.
+If you want to run your test plan and TE ticket with test runner, you should skip --force_serial_run switch.
```

Comment:
so are we allowing jira integration?

Reply:
Currently for internal users only. We will need to modify this later. Let me add that way in the bracket to clarify.

Reply:
This is a bigger discussion topic and needs more modifications. Maybe even to the way the tests are executed. Will differ this activity for now.

Reply:
So do we want to keep references for testrunner as it needs TP/TE?